frontend/src/App.vue

```diff
@@ -44,11 +44,18 @@ export default {
 </script>
 
 <style>
+
+@font-face {
+  font-family: "Core Sans C";
+  src: url("localhost:8080/frontend/src/styles/fonts/CoreSansC-45Regular.eot");
+  src: url("localhost:8080/frontend/src/styles/fonts/CoreSansC-45Regular.woff") format("woff"),
+}
+
```

Comment:
Maybe this should be imported from an API rather than from a local file. The API token could be saved as an environment variables on GitHub and Docker